appveyor.yml

```diff
@@ -44,7 +51,7 @@ install:
   - choco install swig --version 3.0.5 -y
   
   ## Install python-nose for python testing.
-  - C:\Python27-x64\Scripts\pip install nose
+  - %PYTHON_DIR%\Scripts\pip install nose
```

Comment:
Thanks. I found [this page](http://help.appveyor.com/discussions/problems/2869-appdata-causes-error-parsing-appveyoryml-while-scanning-for-the-next-token-find-character-that-cannot-start-any-token), which uses single-quotes, but I'll try double-quotes… must be twice as good.